src/test/java/uk/gov/digital/ho/pttg/testrunner/SmokeTestsServiceTest.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.digital.ho.pttg.testrunner;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import uk.gov.digital.ho.pttg.api.SmokeTestsResult;
+import uk.gov.digital.ho.pttg.testrunner.domain.Applicant;
+import uk.gov.digital.ho.pttg.testrunner.domain.FinancialStatusRequest;
+
+import java.time.LocalDate;
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.springframework.http.HttpStatus.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SmokeTestsServiceTest {
+
+    @Mock
+    private IpsClient mockIpsClient;
+
+    private SmokeTestsService service;
+    private ArgumentCaptor<FinancialStatusRequest> requestCaptor;
+
+    @Before
+    public void setUp() {
+        service = new SmokeTestsService(mockIpsClient);
+        requestCaptor = ArgumentCaptor.forClass(FinancialStatusRequest.class);
+    }
+
+    @Test
+    public void runSmokeTests_always_callIpsWithTestRequest() {
+        service.runSmokeTests();
+
+        then(mockIpsClient).should().sendFinancialStatusRequest(requestCaptor.capture());
+        FinancialStatusRequest expectedRequest = new FinancialStatusRequest(
+                Collections.singletonList(new Applicant("smoke", "tests", LocalDate.now(), "AA000000A")),
```

Comment:
This probably works providing the test doesn't span 23:59->00:00

Reply:
Very unlikely that anybody's going to run the unit tests at that time but I'll try to sort anyway!

Reply:
Fixed